README.md

```diff
@@ -1,14 +1,14 @@
-# Default starter for Gridsome
+# Build
 
-This is the project you get when you run `gridsome create new-project`.
+`npm run build`
 
-### 1. Install Gridsome CLI tool if you don't have
+# Develop
 
-`npm install --global @gridsome/cli`
+`npm run develop`
```

Comment:
develop and build are normally done with the command `gridsome develop` and `gridsome build`

Reply:
That is only possible when Gridsome is installed globally. `npm run [build|develop]` on the other hand can use the Gridsome version that was installed as a dependency in the `node_modules` folder of the project.